app/controllers/hyrax/batch_edits_controller.rb

```diff
@@ -37,10 +37,8 @@ def check_for_empty!
     end
 
     def destroy_collection
-      batch.each do |doc_id|
-        obj = Hyrax.query_service.find_by_alternate_identifier(alternate_identifier: doc_id, use_valkyrie: false)
-        obj.destroy
-      end
+      objs = Hyrax.query_service.custom_queries.find_many_by_alternate_ids(alternate_ids: batch)
+      objs.map(&:destroy)
```

Comment:
Also, does this actually run the destroy? The previous implementation does `#each`; is `#map`'s lazy access a problem here?

Reply:
The delete worked so I didn't dig into it that far. 

Apparently the wings `pcdm_valkyrie_behavior` sends any missing methods to the AF version of the object. So it's an array of valkyrie objects but it's apparently going through the AF destroy anyway. If we want to destroy it via valkyrie, it would require more work. But no, the lazy access doesn't appear to be a problem. 

In addition, deleting the collections in batch apparently does not remove the member of collections association on the work that was in the collection. This error exists in the current code, and my change made no differences in existing behavior.

Reply:
Running it through the persister "delete" method causes the collections to delete but leaves their access controls. The Generic Work keeps the member of collections relationship to the missing collection in both cases. 

I guess that this isn't ready to change to Valkyrie yet.